okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -754,6 +741,57 @@ private void fixupXOkapiToken(ModuleDescriptor md, MultiMap reqHeaders, MultiMap
     }
   }
 
+  private static void streamHandle(ProxyContext pc, ReadStream<Buffer> readStream,
+                                   WriteStream<Buffer> mainWriteStream,
+                                   List<HttpClientRequest> logWriteStreams) {
```

Comment:
`List<HttpClientRequest>` can be changed to `List<? extends WriteStream<Buffer>>`

Reply:
Instead of handling mainWriteStream and logWriteStreams separately they should be combined into one array or list.